tests/test_captcha_privacy.py

```
@@ -190,13 +208,17 @@ def test_no_sensitive_data_in_error_logs(self, caplog):
 
         # Try to solve and trigger error
         with caplog.at_level(logging.DEBUG):
-            solution = solver_manager.solve_recaptcha_v2("test_site_key", "https://example.com")
+            solution = solver_manager.solve_recaptcha_v2(
```

Ruff (0.14.13)

Local variable `solution` is assigned to but never used

Remove assignment to unused variable `solution`

(F841)